### DISCUSSION TITLE
Adjusting UI scaling and making it not blurry

### DISCUSSION THREAD

**Discussion by HubKing** [WerWolv/ImHex#990] [Get Help] [answered]
I am using 175% Gnome Wayland. Default font is too small. x2 is too large. x1.5 still seems too big. Also the UI is blurry. Can I make the UI not blurry and increase the scaling more finely (like 1.3)?

x1.0 or native

![image](https://user-images.githubusercontent.com/10900489/226802373-cb01ad80-1db2-466b-8da3-f55ba3ec02df.png)

x2.0

![image](https://user-images.githubusercontent.com/10900489/226802521-0e7e2916-a3a7-40a8-8bf4-74a1611226f5.png)

x1.5

![image](https://user-images.githubusercontent.com/10900489/226802675-5a796061-d660-49a7-a855-02340cdf257b.png)

**Comment by WerWolv** [comment, marked as answer]
Hey
The default font that ImHex uses is a pixel-perfect bitmap font. Because of that, scaling it at non-integer intervals causes it to become blurry. I improved that behaviour a lot in the latest nightly however I'd suggest you to use either 1.0x or 1.5x and then load a custom font in the settigngs and set its font size to something you like. This can be any TTF or OTF font which also scales more nicely in non-integer levels

**Reply by HubKing** [reply]
Is "pixel-perfect" meaningful when Linux is using desktop scaling? As far as I know, on Windows, apps can sort-of opt out of scaling (and even the user can choose to do that in the compatibility settings), but I am not sure if that is possible in Linux Wayland. If Linux apps cannot opt out of the scaling that the system performs, even if the app renders itself in  a "pixel-perfect" manner in its virtual screen space, the OS (or should I say, the desktop environment, in Linux?) would resize that anyway, so it seems futile.
